### PR TITLE
Update environment to have minimal dependencies.

### PR DESCRIPTION
**Pull request by deeplycloudy**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -6,13 +6,14 @@ dependencies:
...
-- ipython
-- matplotlib
+- pip
+- pyclipper
 - pyproj
 - python=3.6
-- six
\ No newline at end of file
+- scipy
+- six
```
Comment (djhoese) [inline environment.yml]: Is `six` needed? This is a python 3-only package, right?

Comment (dopplershift) [reply environment.yml]: More importantly, it's not a dependency of this code base, so IMO it shouldn't be listed. Let the other packages provide their dependencies.

**Comment by deeplycloudy** [comment]
@djhoese It’s needed for lmatools, which still uses six. I haven’t gone through to update lmatools to 3 only.

**Comment by deeplycloudy** [comment]
@dopplershit would I do that with a requirements.txt in lmatools?

**Comment by dopplershift** [comment]
It’d be in `setup.py` or `setup.cfg` in lmatools.

**Comment by djhoese** [comment]
See https://github.com/deeplycloudy/lmatools/pull/26 for python 3-ification of lmatools.

**Comment by deeplycloudy** [comment]
@djhoese following the recommendation of @dopplershift I updated lmatools to list its requirements, and added that to your branch that removes python 2 support from lmatools. So, once deeplycloudy/lmatools#26 is merged this PR LGTM.

File: environment.yml (hunk 1 above)
Comment (deeplycloudy) [reply environment.yml]: Just pushed a commit to remove six, as well as pyproj, per the principles articulated above.

**Comment by deeplycloudy** [comment]
@djhoese after today's changes do you need to do any further testing with this branch before I merge into ugf-newgrid?

**Pull request merged**